backend/typescript/swagger.yml

```diff
@@ -540,13 +325,13 @@ paths:
           description: Invalid request
 
                 
-  /users/{userId}:

```

Comment:
@Tahmeed27 Should these endpoints work? It gives me an error which comes from isAuthorizedByRole in auth.ts b/c the accessToken is null and role is empty for me.. but not sure if this is supposed to happen?